config/src/main/java/com/alibaba/nacos/config/server/controller/ConfigController.java

```diff
@@ -690,11 +691,19 @@ public GroupkeyListenserStatus getListeners(@RequestParam("dataId") String dataI
         return ResultBuilder.buildSuccessResult("Clone Completed Successfully", saveResult);
     }
     
+    /**
+     * Treat the tenant parameters with values of "public" and "null" as an empty string.
+     * @author klw(213539@qq.com)
+     * 2020/10/12 15:59
+     * @param tenant tenant
+     * @return java.lang.String
```

Comment:
return annotation should describe what content method return, only type is not useful